### ISSUE TITLE
Varcem startup problem

### ISSUE THREAD

**Issue by Fenix770** [VARCem/VARCem#44]
Hello!

Sorry for writing again about this issue!

I installed Varcem 1.6.3 (win 32 installer) and I found the emulator crashes at startup. I installed all the components except the virtual machines, because I wanted to save some disk space. 
I don not know If i am some dll file, in Windows, is missing, but 86box works. 

PD: I remebered having the same issue with some  automated builds  of 86box last year, but this year all builds I tried worked.

I attach the  screenshot and the dump file here 

![image 5](https://user-images.githubusercontent.com/31600563/45006255-648ebd00-afce-11e8-819f-8f9402290077.png)

[20180904-01-43-41-987.zip](https://github.com/VARCem/VARCem/files/2346938/20180904-01-43-41-987.zip)

**Comment by waltje** [comment]
Hi,

What operating system is this running on?

I just tried on a clean Windows 7 (x64) and Windows XP (x86) and no problems...

--waltje

--------------------------------------------
On Mon, 9/3/18, Fenix770 <notifications@github.com> wrote:

 Subject: [VARCem/VARCem] Varcem startup problem (#44)
 To: "VARCem/VARCem" <VARCem@noreply.github.com>
 Cc: "Subscribed" <subscribed@noreply.github.com>
 Date: Monday, September 3, 2018, 10:20 PM
 
 Hello!
 Sorry for writing again about this issue!
 I installed Varcem 1.6.3 (win 32 installer) and I found
 the emulator crashes at startup. I installed all the
 components except the virtual machines, because I wanted to
 save some disk space.
 
 I don not know If i am some dll file, in Windows, is
 missing, but 86box works.
 PD: I remebered having the same issue with some 
 automated builds  of 86box last year, but this year all
 builds I tried worked.
 I attach the  screenshot and the dump file here
 
 20180904-01-43-41-987.zip
 
 —
 You are receiving this because you are subscribed to this
 thread.
 Reply to this email directly, view it on
 GitHub, or mute
 the thread.
